agent/handlers/task_server_setup_test.go

```diff
@@ -476,6 +476,8 @@ var (
 
 	ipSources = []string{"52.95.154.1", "52.95.154.2"}
 
+	ipSourcesToFilter = []string{"8.8.8.8"}
```

Comment:
I think we should also add test cases to check that invalid `SourcesToFilter` values cause a validation failure.

Reply:
This is added to handler_test.go in the shared library. In that new test case we pass in `"SourcesToFilter": []string{"52.95.154.0/33"},`, which will cause a validation failure

Reply:
Thanks. I should have looked at that file.